projects/BFDR/BirthRecord_submissionProperties.cs

```diff
@@ -3950,6 +3953,693 @@ public string FatherEthnicityLiteral
+        /// </example>
+        [Property("DateOfLastOtherPregnancyOutcomeMonth", Property.Types.Int32, "Date of Last Other Pregnancy Outcome", "Date of last other pregnancy outcome.", false, IGURL.ObservationDateOfLastOtherPregnancyOutcome, false, 34)]
+        [PropertyParam("dateOfLastOtherPregnancyOutcomeMonth", "The month of the last other pregnancy outcome.")]
+        [FHIRPath("Bundle.entry.resource.where($this is Observation).where(code.coding.code='68500-8')", "")]
+        public int? DateOfLastOtherPregnancyOutcomeMonth
+        {
+            get
+            {
+                Observation obs = GetObservation("68500-8");
+                if (obs != null)
+                {
+                    return GetDateFragmentOrPartialDate(obs.Value, PartialDateMonthUrl);    
+                }
+                return null;
+            }
+            set
+            {
+                if (value == null)
+                {
+                    return;
+                }
+                Observation obs = GetObservation("68500-8");
+                if (obs == null)
+                {
+                    obs = GetOrCreateObservation("68500-8", CodeSystems.LOINC, BFDR.ProfileURL.ObservationDateOfLastOtherPregnancyOutcome, DATE_OF_LAST_OTHER_PREGNANCY_OUTCOME, Mother.Id);
+                    obs.Value = new FhirDateTime();
+                }
+                if (obs.Value.Extension.Find(ext => ext.Url == PartialDateUrl) == null)
+                {
+                    // must be created here to use the override Extension URL
+                    obs.Value.Extension.Add(NewBlankPartialDateTimeExtension(false));
+                }
+                SetPartialDate(obs.Value.Extension.Find(ext => ext.Url == PartialDateUrl), PartialDateMonthUrl, value);
+                return;
+            }
+        }
+
+        /// <summary>Date Of Last Other Pregnancy Outcome year</summary>
+        /// <value>the date of the Mother's last other pregnancy outcome
+        /// </value>
+        /// <example>
+        /// <para>// Setter:</para>
+        /// <para>ExampleBirthRecord.DateOfLastOtherPregnancyOutcomeYear = 2022;</para>
+        /// <para>// Getter:</para>
+        /// <para>Console.WriteLine($"Date of Last Other Pregnancy Outcome:: {ExampleBirthRecord.DateOfLastOtherPregnancyOutcomeYear}");</para>
+        /// </example>
+        [Property("DateOfLastOtherPregnancyOutcomeYear", Property.Types.Int32, "Date of Last Other Pregnancy Outcome", "Date of last other pregnancy outcome.", false, IGURL.ObservationDateOfLastOtherPregnancyOutcome, false, 34)]
+        [PropertyParam("dateOfLastOtherPregnancyOutcomeYear", "The year of the last other pregnancy outcome.")]
+        [FHIRPath("Bundle.entry.resource.where($this is Observation).where(code.coding.code='68500-8')", "")]
+        public int? DateOfLastOtherPregnancyOutcomeYear
```

Comment:
Same question: Is there a reason why there's a month/year `DateOfLastOtherPregnancyOutcome*` but not a day?

Reply:
In IJE there was only a month and year field, but it would make sense to allow day data in FHIR. I will add it in.